src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -119,41 +119,26 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
-  "WorkflowComponent" should "let you modify Workflows within a submission" in withDefaultTestDatabase {
```

Comment:
i couldn't get a test working that would throw the RawlsConcurrentModificationException error. should that be a requirement for this PR to go through?